### PR TITLE
implementing the AS_IS option

### PR DESCRIPTION
**Pull request by zdanowiczkonrad**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -61,34 +62,36 @@ function roundToPlace(number, place) {
...
-    const fullPriceInNormalFormat = parseFloat(
+
+    const roundedPrice = parseFloat(
             (opts.arithmeticalRounding ? roundToPlace(number, opts.decimalsDigits) : +number)
-        .toString())
-        .toFixed(opts.showDecimals === SHOW_DECIMALS.NEVER ? 0 : opts.decimalsDigits);
+        .toString());
+    const fullPriceInNormalFormat = opts.showDecimals !== SHOW_DECIMALS.AS_IS ?
+      roundedPrice.toFixed(opts.showDecimals === SHOW_DECIMALS.NEVER ? 0 : opts.decimalsDigits) :
+      roundedPrice.toString();
```
Comment (zdanowiczkonrad) [inline index.js]: `.toString` is redundant here.

File: test.js [hunk 2]
```diff
@@ -23,6 +23,8 @@ assert(format(123.456, { decimalsDigits: 3 }), '123.456', 'Decimals should follo
 assert(format(123.4), '123.40', 'Decimals should follow the decimal precision for input that has less decimal digits');
 assert(format(123.45, { decimalsDigits: 5 }), '123.45000', 'Custom decimal digits should work for larger decimal and padding');
 assert(format(123, { decimalsDigits: 5, showDecimals: 'ALWAYS' }), '123.00000', 'Custom decimal digits should be respected when no decimals but showing decimals is forced');
+assert(format(123333.4567, { showDecimals: 'AS_IS' }), '123,333.4567', 'AS_IS does the formating but does not round the decimal places');
+assert(format(123.123, { decimalsDigits: 5, showDecimals: 'AS_IS' }), '123.123', 'AS_IS also ignores any decimalsDigits setting');
```
Comment (zdanowiczkonrad) [inline test.js:30]: can you also add test cases for 123.0, 123.1 and 123?

**Review by zdanowiczkonrad — COMMENTED** [review]
Very good changes @MartinHDB, thanks for your contribution! I'd only ask you to update the readme to include the new `AS_IS` option and please add missing test cases as well as remove redundant `toString` as noted in the comment and we're ready to go! feel free also to bump the version when done with `npm version patch` command and pushing these changes to the remote.

**Comment by zdanowiczkonrad** [comment]
Adds support fo floating decimals parsing as of #1 .

**Comment by gdarai** [comment]
Ok, readme updated. I also added the three test cases you asked for and had to tweak the code a little to handle the 123.0 case. It actually has to be in string, because the float 123.0 is read as 123... but that does not matter, that is I suppose expected behaviour.

The toString() is still there, because the ```fullPriceInNormalFormat``` has to be string no matter if number is string or number. So it is still there, even though this part is also again a little bit tweaked.

**Comment by gdarai** [comment]
@zdanowiczkonrad, When you will have time, the PR is ready to be checked again.

**Comment by zdanowiczkonrad** [comment]
Excellent stuff! Thanks!

**Pull request merged**